webpack.config.js

```diff
@@ -42,10 +44,10 @@ module.exports = {
     chunkFilename: 'js/[name].js',
   },
   optimization: {
-    minimize: true,
+    minimize: isProduction,
     minimizer: [
       new TerserPlugin({
-        sourceMap: true,
+        sourceMap: isProduction,
```

Comment:
I think it'd be helpful to have the source map available while developing so it may be better to leave this as true.
```suggestion
        sourceMap: true,
```

Reply:
Indeed it's good to have them in dev but as minification is not used there, I think the option need to be set on `babel-loader` instead, trying that.

Reply:
It seems source maps are always enabled in development to me, it seems to make no difference if I enable it on babel-loader or not. Anyways, I'll set the option just to be sure.